ui/helpers/urlHelper.js

```diff
@@ -36,8 +36,8 @@ export const getLogViewerUrl = (job_id, repoName, line_number) => {
 // if we can't find "repo" on the url (like in perfherder) they don't
 // care WHICH repo it is, just need one because of Bug 1441938 (see above).
 // So we default to mozilla-inbound.
-export const getProjectUrl = (uri) => {
-  const repo = getUrlParam("repo") || 'mozilla-inbound';
+export const getProjectUrl = (uri, repoName) => {
+  const repo = getUrlParam("repo") || repoName || 'mozilla-inbound';
```

Comment:
Is `repoName` being set the common case? If so, should it go before `getUrlParam`?

Reply:
I was going to push back on this, but I think you make a good point.  :)  If you pass in a repoName, you wouldn't want it to ignore that if the param happens to be in the URL.  If you pass in ``repoName`` you mean business, sir!  I'll switch it.  :)